easy-ui-react/src/ToggleCard/ToggleCard.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { ReactNode, useMemo } from "react";
+import { Card } from "../Card";
+import { HorizontalStack } from "../HorizontalStack";
+import {
+  flattenChildren,
+  getDisplayNameFromReactNode,
+} from "../utilities/react";
+
+import styles from "./ToggleCard.module.scss";
+
+export type ToggleCardProps = {
+  /**
+   * The children of the <ToggleCard> element. Should render
+   * `<ToggleCard.Header>` and `<ToggleCard.Body>`
+   */
+  children: ReactNode;
+};
+
+/**
+ * A `<ToggleCard />` is a styled container with an interactive
+ * header featuring a toggle control.
+ *
+ * @remarks
+ * It is a simple compound component consisting of `<ToggleCard.Header />`
+ * and `<ToggleCard.Body />`.
+ *
+ * `<ToggleCard.Header />` should handle the rendering for the `Toggle`
+ * control and the content alongside it.
+ *
+ * `<ToggleCard.Body />` is responsible for rendering the body content.
+ *
+ * @example
+ * _Controlled:_
+ * ```tsx
+ * <ToggleCard>
+ *  <ToggleCard.Header>
+ *    <Text id="some id" variant="subtitle1" color="primary.900">
+ *      header
+ *    </Text>
+ *    <Toggle
+ *      aria-labelledby="some id"
+ *      isSelected={isSelected}
+ *      onChange={(isSelected) => setIsSelected(isSelected)}
+ *     />
+ *  </ToggleCard.Header>
+ *  <ToggleCard.Body>body</ToggleCard.Body>
+ * </ToggleCard>
+ * ```
+ *
+ * @example
+ * _Disabled:_
+ * ```tsx
+ * <ToggleCard>
+ *  <ToggleCard.Header>
+ *    <Text id="some id" variant="subtitle1" color="primary.900">
+ *      header
+ *    </Text>
+ *    <Toggle aria-labelledby="some id" isDisabled />
+ *  </ToggleCard.Header>
+ *  <ToggleCard.Body>body</ToggleCard.Body>
+ * </ToggleCard>
+ * ```
+ *
+ * @example
+ * _Read-Only:_
+ * ```tsx
+ * <ToggleCard>
+ *  <ToggleCard.Header>
+ *    <Text id="some id" variant="subtitle1" color="primary.900">
+ *      header
+ *    </Text>
+ *    <Toggle aria-labelledby="some id" isReadOnly />
+ *  </ToggleCard.Header>
+ *  <ToggleCard.Body>body</ToggleCard.Body>
+ * </ToggleCard>
+ * ```
+ */
+export function ToggleCard(props: ToggleCardProps) {
+  const { children } = props;
+
+  return <Card.Container>{children}</Card.Container>;
```

Comment:
looking at the design seems we need to accept a `borderRadius` prop here with our token values similar to boxShadow. would it make sense to include in this task or follow up later?

Reply:
oh interesting, didnt even catch that the border radius is totally different, ill add it to the same ticket